components/html-block/test/html-block.test.js

```diff
@@ -143,7 +144,7 @@ describe('d2l-html-block', () => {
 		expect(spans[1].innerHTML).to.equal('2');
 	});
 
-	it('should do async replacements', async() => {
+	it.only('should do async replacements', async() => {
```

Comment:
Is this `only` intentional/temp?

Reply:
Shoot, no. Definitely just slipped in by accident from my debugging.